pulsar-broker/src/test/java/org/apache/pulsar/client/api/BrokerServiceLookupTest.java

```diff
@@ -920,7 +913,7 @@ public void testSplitUnloadLookupTest() throws Exception {
      *
      * @throws Exception
      */
-    @Test(timeOut = 10000)
+    @Test(timeOut = 20000)
```

Comment:
do we really need these timeout ?

Reply:
I left it there since it already pre-existed. :) Perhaps they could be removed, but since the CI is still so flaky, I'd rather not revisit the PR just to remove the timeout...

Reply:
Yes,  no need to change now, but in general most of the timeouts were added before we had the global default timeout for the tests. Others then kept creeping in after that.

As general rule, I think we should prefer just relying on the global timeout.